packages/cli/src/commands/server/middleware/getDevToolsMiddleware.js

```diff
@@ -6,29 +6,29 @@
  *
  * @format
  */
-import launchChrome from '../launchChrome';
 import {logger} from '@react-native-community/cli-tools';
 import {exec} from 'child_process';
+import launchDebugger from './../launchDebugger';
```

Comment:
```suggestion
import launchDebugger from '../launchDebugger';
```

Reply:
I don't see why it shouldn't but nothing wrong in testing. I use Brave and it works without an issue but that uses Chrome DevTools. I will test this on Firefox etc. These are essentially websockets and http requests so in theory it should work